src/orion/benchmark/assessment/warm_start_efficiency.py

```diff
@@ -0,0 +1,253 @@
+from orion.benchmark.assessment.base import BaseAssess
+from orion.plotting.base import regrets
+from typing import List, Tuple, Dict
+from orion.core.worker.experiment import Experiment
+from orion.client import ExperimentClient
+from plotly.graph_objects import Figure
+from .averageresult import AverageResult, regrets
+from .averagerank import AverageRank, rankings
+from collections import defaultdict
+import plotly.graph_objs as go
+from orion.core.worker.trial import Trial
+import plotly
+from pandas.core.groupby import DataFrameGroupBy
+from typing import Callable
+import pandas as pd
+import orion.analysis
+
+WarmStartExperimentsTuple = Tuple[ExperimentClient, ExperimentClient, ExperimentClient]
+
+
+class WarmStartEfficiency(BaseAssess):
+    """
+    TODO: Evaluate the 'warm start efficiency' (objective value) for each search algorithm
```

Comment:
That's for another assessment object? Should be in an issue for another PR I guess?

Reply:
Well, actually, would you mind moving the warm-starting benchmark stuff to another PR so that we can first focus on warm-starting itself?

Reply:
Good idea, however I wouldn't know how to create the PR for the warm-start benchmark while this one isn't integrated, since it depends on it. Should I create that PR against this branch now? or against develop after this is merged?